### ISSUE TITLE
Inappropriate content functionality

### ISSUE THREAD

**Issue by marilynfletcher** [MAPC/bostonparks#87]
Tyrel -- are you partway through this?  I can pick it up if you want.  Marilyn

**Comment by tyrelsouza** [comment]
It functions correctly, there just is no mail server on the dev server so it doesn't "work", but all the code is correct when there would be a mail server running as it just uses django's sendmail.

**Comment by marilynfletcher** [comment]
Okay, I'll leave this issue up as reminder.

**Comment by cspanring** [comment]
So we just need to add email settings? 

EMAIL_HOST, EMAIL_PORT, EMAIL_HOST_USER, EMAIL_HOST_PASSWORD, EMAIL_USE_TLS

**Comment by tyrelsouza** [comment]
Yes. And make sure your local_settings ADMINS  tuple is correctly setup 
the same way that it's setup in settings.py.

**Comment by tyrelsouza** [comment]
This emails out, and works pretty well.

**Comment by cspanring** [comment]
Please leave it open as a reminder for me to update the email settings on our production server.

**Comment by tyrelsouza** [comment]
Alright, my bad

--ts
